solr/contrib/prometheus-exporter/conf/solr-exporter-config.xml

```diff
@@ -315,88 +477,22 @@
             node metrics
           -->
           <str>
-            .metrics["solr.node"] | to_entries | .[] | select(.key | endswith(".clientErrors")) as $object |
-            $object.key | split(".")[0] as $category |
-            $object.key | split(".")[1] as $handler |
-            $object.value.count as $value |
-            {
-              name         : "solr_metrics_node_client_errors_total",
-              type         : "COUNTER",
-              help         : "See following URL: https://lucene.apache.org/solr/guide/metrics-reporting.html",
-              label_names  : ["category", "handler"],
-              label_values : [$category, $handler],
-              value        : $value
-            }
+            $jq:node(client_errors_total, select(.key | endswith(".clientErrors")), count)
           </str>
           <str>
-            .metrics["solr.node"] | to_entries | .[] | select(.key | endswith(".clientErrors")) as $object |
-            $object.key | split(".")[0] as $category |
-            $object.key | split(".")[1] as $handler |
-            $object.value.count as $value |
-            {
-              name         : "solr_metrics_node_errors_total",
-              type         : "COUNTER",
-              help         : "See following URL: https://lucene.apache.org/solr/guide/metrics-reporting.html",
-              label_names  : ["category", "handler"],
-              label_values : [$category, $handler],
-              value        : $value
-            }
+            $jq:node(errors_total, select(.key | endswith(".errors")), count)
           </str>
           <str>
-            .metrics["solr.node"] | to_entries | .[] | select(.key | endswith(".requestTimes")) as $object |
-            $object.key | split(".")[0] as $category |
-            $object.key | split(".")[1] as $handler |
-            $object.value.count as $value |
-            {
-              name         : "solr_metrics_node_requests_total",
-              type         : "COUNTER",
-              help         : "See following URL: https://lucene.apache.org/solr/guide/metrics-reporting.html",
-              label_names  : ["category", "handler"],
-              label_values : [$category, $handler],
-              value        : $value
-            }
+            $jq:node(requests_total, select(.key | endswith(".local.requestTimes")), count)
```

Comment:
@thelabdude I noticed here you added a ".local." when it wasn't there before.  Why?  And for that matter, maybe we needn't bother publishing this particular metric at all; I'm skeptical of the utility.

Reply:
@dsmiley it's to support the query charts showing core-level query metrics vs. top-level distributed query metrics added in this PR. I like knowing if there's an imbalance of core-level query requests going to certain replicas or if the load across all of my replicas is balanced. So you're skeptical but you haven't said why exactly? You want to change, then change it.

Reply:
fwiw ~ have you actually looked at the charts added in this PR in Grafana with query load running? If there's a problem there, then let's fix it and move forward but rehashing old decisions seems unproductive at this point.

Reply:
I meant to comment on the "totalTime" metric w.r.t. it's usefulness; sorry for the confusion.  It's some massive number of course... it'd need to be divided by something else to be useful?  Also, totalTime is in nanoseconds lately!  https://issues.apache.org/jira/browse/SOLR-16073

I understand the overarching objective of top-level vs core-level -- makes sense. 
I'm a bit unclear on the distinction between the node level "$jq:node" metrics, and the "Local (non-distrib) query metrics", both of which are using ".local.".

RE Grafana; I haven't seen our official one in use live.  I use our own/custom one at work.